internal/nginx/nginx.go

```diff
@@ -402,15 +402,10 @@ func (nginx *Controller) UpdateMainConfigFile(cfg []byte) {
 	}
 
 	if !nginx.local {
-		w, err := os.Create(filename)
+		err := createFileAndWrite(filename, cfg)
 		if err != nil {
-			glog.Fatalf("Failed to open %v: %v", filename, err)
+			glog.Fatalf("%v", err.Error())
```

Comment:
It is better to add more context to the log message:
```go
glog.Fatalf("Failed to write NGINX conf: %v", err)
```
Additionally, I don't see why we need to use `err.Error())` instead of just  `err`

Reply:
@pleshakov  ok I will modify in accordance with you suggest